pythonforandroid/recipes/python3/patches/py3.11.4_fix_cortex_a8.patch

```diff
@@ -0,0 +1,11 @@
+--- Python-3.11.4/configure	2023-06-07 03:30:27.000000000 +0530
++++ Python-3.11.4.mod/configure	2023-07-08 11:51:11.031135937 +0530
+@@ -6994,7 +6994,7 @@
+ printf "%s\n" "$_arm_arch" >&6; }
+   if test "$_arm_arch" = 7; then
+     BASECFLAGS="${BASECFLAGS} -mfloat-abi=softfp -mfpu=vfpv3-d16"
+-    LDFLAGS="${LDFLAGS} -march=armv7-a -Wl,--fix-cortex-a8"
```

Comment:
`-Wl,--fix-cortex-a8` is flags are into CPython codebase since (at least) `~7 years`, can you describe why should be removed?

Reply:
Same thing is done in every other python version in p4a?

Reply:
Same thing is done in every other python version in p4a?

Reply:
Sorry for the noise. Reading patches is sometimes really difficult (diff in a diff). Got it.